hack/vendor/olm-test-script/README.md

```diff
@@ -0,0 +1,29 @@
+# OLM Test Script
+
+## Description
+This script can be used to test install your operator's manifest file in an OCP cluster
+
+### Variables
```

Comment:
operator image env var?

Reply:
All items related to `vendor` are meant to be straight copies from the original repo in order foster the original repo being maintained and the vedor here being periodically updated.  Seems worthy to try and maintain that

Reply:
ok.